addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -182,7 +182,7 @@ test("chat window: basic rendering", async () => {
         text: "There are no messages in this conversation.",
     });
     await click("[title='Open Actions Menu']");
-    await contains(".o-mail-ChatWindow-command", { count: 13 });
+    await contains(".o-mail-ChatWindow-command", { count: 14 });
```

Comment:
Whenever this counter is changed, the listing below should be adapted. There's a new thread action (view/create subthreads), explicitly `contains()` it below!)